src/scene/create.js

```diff
@@ -0,0 +1,47 @@
+import Phaser from 'phaser'
+import {WORLD} from '../world'
+import {GAME_CONFIG, CONFIG} from '../config'
+import {create as createFeatures} from '../features'
+import {SPRITES} from '../sprites'
+
+const KEYS = Phaser.Input.Keyboard.KeyCodes
+// console.log(Object.keys(KEYS))
+// TODO: figure out why D doesn't work. (tried WASD config, but the D key was furked)
```

Comment:
This is weird -- other people should try testing this, but if you set `W`/`A`/`S`/`D` keys below, they all work except the `D`? I tried a couple different things and resolved to deal with the problem at a later date.